mesheryctl/pkg/utils/helpers.go

```diff
@@ -950,6 +950,15 @@ func SetOverrideValues(ctx *config.Context, mesheryImageVersion string) map[stri
 		}
 	}
 
+	// disable the operator
+	if ctx.GetDisableOperator() {
+		if valueOverrides["env"] == nil {
+			valueOverrides["env"] = map[string]interface{}{}
+		}
+		envOverrides := valueOverrides["env"].(map[string]interface{})
```

Comment:
Why cant we use the below statement like the other env variables? 
```
        valueOverrides["env"] = map[string]interface{}{
			"DISABLE_OPERATOR": true,
		}
```

Reply:
@hexxdump It is written like this to prevent overriding of the PROVIDER value -

<img width="585" alt="image" src="https://user-images.githubusercontent.com/76054921/222928159-42d27ad8-b888-47a9-86df-399803fc5e72.png">

Reply:
By looking at lines 933-951, that does not seem probable. As per your comment, components, image and provider must overwrite each other's values.

Reply:
@hexxdump In 933-951, the values are written to different keys hence overwriting doesn't occur. Image is written to `image` and provider is written to `env`. Whereas for assigning `PROVIDER` and `DISABLE_OPERATOR` the value should be the written to the same `env` key.

Reply:
> Why cant we use the below statement like the other env variables?
> 
> ```
>         valueOverrides["env"] = map[string]interface{}{
> 			"DISABLE_OPERATOR": true,
> 		}
> ```

If it is written like above the lines 948-950 have no effect.

Reply:
> @hexxdump In 933-951, the values are written to different keys hence overwriting doesn't occur. Image is written to `image` and provider is written to `env`. Whereas for assigning `PROVIDER` and `DISABLE_OPERATOR` the value should be the written to the same `env` key.

@abdullah1308 , got it! Thanks for the clarification.